src/python/impactx/dashboard/Input/distributionParameters/distributionMain.py

```diff
@@ -177,19 +177,27 @@ def card_content(self):
                         v_for="(parameter, index) in selected_distribution_parameters",
                         cols=4,
                     ):
-                        vuetify.VTextField(
-                            label=("parameter.parameter_name",),
-                            v_model=("parameter.parameter_default_value",),
-                            suffix=("parameter.parameter_units",),
-                            update_modelValue=(
-                                ctrl.updateDistributionParameters,
-                                "[parameter.parameter_name, $event, parameter.parameter_type]",
-                            ),
-                            error_messages=("parameter.parameter_error_message",),
-                            type="number",
-                            step=("parameter.parameter_step",),
-                            __properties=["step"],
-                            density="compact",
-                            variant="underlined",
-                            hide_details="auto",
-                        )
+                        with vuetify.VTooltip(
+                            location="top",
+                            text=("all_tooltips[parameter.parameter_name]",),
```

Comment:
I'm surprised that this works and you don't need 
```py
text=(f"all_tooltips[{parameter.parameter_name}]",),
```
instead. What am I missing?